### DISCUSSION TITLE
How to use counts with commands in config?

### DISCUSSION THREAD

**Discussion by giomf** [helix-editor/helix#9601] [General]
Hi

is it possible to use counts with commands in config?
For example if I want to navigate a little faster without the use of `half_page_down`.
I was trying something like this but its not working. The Idea is to move three lines down:
```
 J = [ "3", "move_visual_line_down" ]
```
Is it possible in some way?

**Comment by the-mikedavis** [comment]
This is not currently possible, it needs https://github.com/helix-editor/helix/issues/1383

**Reply by giomf** [reply]
Thanks for clarification.

**Comment by pantos9000** [comment]
I tried the same and was able to work around it with:
```
J = ["move_visual_line_up", "move_visual_line_up", "move_visual_line_up", "move_visual_line_up", "move_visual_line_up"]
```

This is very limited, but maybe it helps.

**Reply by giomf** [reply]
Yes I also use this approach.
Works for now but definitely needs a better solution.
